Client/gui/CGUI_Impl.cpp

```diff
@@ -496,12 +496,47 @@ bool CGUI_Impl::IsCursorEnabled()
 
 void CGUI_Impl::SetCursorAlpha(float fAlpha, bool bOnlyCurrentServer)
 {
-    CEGUI::MouseCursor::getSingleton().setAlpha(fAlpha);
+    CEGUI::MouseCursor::getSingleton().setAlpha(m_fCurrentServerCursorRed, m_fCurrentServerCursorGreen, m_fCurrentServerCursorBlue, fAlpha);
 
     if (bOnlyCurrentServer)
         SetCurrentServerCursorAlpha(fAlpha);
 }
 
+void CGUI_Impl::SetCursorColor(float r, float g, float b, float alpha) noexcept
+{
+    CEGUI::MouseCursor::getSingleton().setColor(r, g, b, alpha);
```

Comment:
If a function calls another function that is not noexcept, then it should also not be noexcept, because this contradicts the assumptions of that keyword